Accessibility Statement and Palette update

nikola-naydenov-hmcts opened the pull request.

kremi reviewed (commented):
Could we please also update the line that states:
> This statement was prepared on 14 September 2021. It was last reviewed on 18 July 2022.

It should say:
> This statement was prepared on 14 September 2021. It was last reviewed on 1st August 2023.

nikola-naydenov-hmcts commented:
> Could we please also update the line that states:
> 
> > This statement was prepared on 14 September 2021. It was last reviewed on 18 July 2022.
> 
> It should say:
> 
> > This statement was prepared on 14 September 2021. It was last reviewed on 1st August 2023.

I updated it yesterday so I changed it to 31 July 2023, also saves me translating the welsh
